python/ray/tests/conftest_docker.py

```diff
@@ -65,7 +66,12 @@ def print_logs(self):
             print(content.decode())
 
 
-gcs_network = network(driver="bridge")
+ipam_config = docker.types.IPAMConfig(
```

Comment:
can you comment how this works? (how the poor networking thing works. how often it happens etc.)

Btw, this is very nice to have such config.. will be very useful for unit test